frontends/web/src/locales/en/app.json

```diff
@@ -814,7 +814,7 @@
     "coincontrol": {
       "address": "Address",
       "outpoint": "Outpoint",
-      "title": "Send from Output"
+      "title": "Send from output"
     },
     "confirm": {
       "selected-coins": "Selected Coins",
```

Comment:
⚠️

And below:
- Confirm transaction
- Converted fee (twice)
- Optional data